incubator/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -887,6 +884,7 @@ private int decodeConnectPayload(
         int progress = offset;
 
         progress = server.onDecodeConnectPayload(traceId, authorization, buffer, progress, limit);
+        decodableRemainingBytes -= progress - offset;
```

Comment:
`decodableRemainingBytes` has to be stored on `server` instead of the factory, so that multiple `server` streams can proceed in parallel without interfering, makes sense?